.gitignore

```diff
@@ -11,6 +11,7 @@ analysis-dashboard
 *-pom
 *-plugin
 echarts-build-trends
+docker/volumes
```

Comment:
It seems that I removed that by accident in my previous PR. I restored the old version, so this should not be required anymore. 

```suggestion
```